src/main/java/automation/testbed/TestbedGrid.java

```diff
@@ -0,0 +1,49 @@
+package automation.testbed;
+
+import automation.Config;
+import org.openqa.selenium.Platform;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.firefox.FirefoxOptions;
+import org.openqa.selenium.remote.Browser;
+import org.openqa.selenium.remote.CapabilityType;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class TestbedGrid extends BaseTestbed
+{
+    @Override
+    public WebDriver createDriver() {
+        String gridHost;
+        String gridPort;
+        DesiredCapabilities caps = new DesiredCapabilities();
+        if ("chrome".equalsIgnoreCase(Config.WEB_BROWSER.value)) {
+            gridHost = Config.SELENIUM_GRID_CHROME_HOST.value;
+            gridPort = Config.SELENIUM_GRID_CHROME_PORT.value;
+            caps.setCapability(ChromeOptions.CAPABILITY, this.getCommonChromeOptions());
+            caps.setCapability(CapabilityType.BROWSER_NAME, Browser.CHROME.browserName());
+            caps.setCapability(CapabilityType.TIMEOUTS, 30);
+            caps.setCapability("se:vncEnabled", true);
+            caps.setCapability(CapabilityType.BROWSER_VERSION, "117.0");
```

Comment:
This line should be commanted because you are using `selenium/standalone-chrome:118.0`